server_admin/topics/identity-broker/first-login-flow.adoc

```diff
@@ -62,7 +62,14 @@ Verify Existing Account By Re-authentication::
   This authenticator is used if email authenticator is disabled or not available (SMTP not configured for realm). It will display a login screen
   where the user needs to authenticate with his password to link their {project_name} account with the Identity provider.
   User can also re-authenticate with some different identity provider, which is already linked to their {project_name} account.
-  You can also force users to use OTP. Otherwise it's optional and used only if OTP is already set for the user account.
+  You can also force users to use OTP. Otherwise it's optional and used only if OTP is already set for the user account. +
+   +
+  This authenticator will behave differently depending on whether the Keycloak user identity has been already established by preceding executions.
+  For example, `Create User If Unique` guarantees that the Keycloak user will be either resolved (matching the external identity), or created otherwise.
+  In this case, login field will be pre-filled and non-editable.
+  However, there are cases where the users neither should be auto-created (e.g. read-only LDAP), nor they could be matched to the external identity (e.g. usernames are different).
```

Comment:
Always use "for example" instead of "e.g."   I would reword as follows: 
However, there are cases where the user should neither be auto-created (for example, a read-only LDAP) nor matched to an external identity (for example, the usernames are different).